### PR TITLE
Added twp modules AAP-31734

### PR DESCRIPTION
**Pull request by annibond**
Added two modules on:
 - Container-based installation of aap 2.5 to 2.x
 - RPM-based installation of aap of 2.5 to 2.x
 for the Ansible Automation Platform guide and updated the master.adoc to include the two titles as.

### CODE REVIEW

File: downstream/modules/platform/proc-update-aap-rpm.adoc [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+* Clustered upgrades require special attention to instance and instance groups before upgrading. Ensure you capture your inventory or instance group details before upgrading. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html-single/configuring_automation_execution/index#controller-clustering[Clustering].
+
+* If you are currently running {EDAcontroller}, it is recommended that you disable all rulebook activations before upgrading to ensure that only new activations run after the upgrade process has completed. This prevents possibilities of orphaned containers running activations from the previous version. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html-single/using_automation_decisions/index#eda-enable-rulebook-activations[Enabling and disabling rulebook activations].
+
+== Choosing and obtaining a {PlatformName}installer
```
Comment (michellemacrh) [inline downstream/modules/platform/proc-update-aap-rpm.adoc:17]: I think you might need a space between {PlatformName}installer

File: downstream/modules/platform/proc-update-aap-rpm.adoc [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+Choose the {PlatformName} installer you need based on your {RHEL} environment internet connectivity. Review the following scenarios and decide on which {PlatformNameShort} installer meets your needs.
+Note
```
Comment (michellemacrh) [inline downstream/modules/platform/proc-update-aap-rpm.adoc:20]: I think the formatting for a note is this:
```
[NOTE]
====
Note content here
====
```

File: downstream/modules/platform/proc-update-aap-rpm.adoc [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+[literal, options="nowrap" subs="+attributes"]
+----
+$ tar xvzf ansible-automation-platform-setup-<latest-version>.tar.gz
+----
++
+*RPM install*
+
+* Install {PlatformNameShort} Installer Package
++
+[literal, options="nowrap" subs="+attributes"]
```
Comment (michellemacrh) [inline downstream/modules/platform/proc-update-aap-rpm.adoc:45]: I think you may need to sanity check code blocks like this to make sure they build locally and look OK :)

File: downstream/modules/platform/proc-update-aap-rpm.adoc [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+. Extract the files:
++
+[literal, options="nowrap" subs="+attributes"]
+----
++$ tar xvzf ansible-automation-platform-setup-bundle-<latest-version>.tar.gz
+----
+
+== Setting up the inventory file
+
+Before upgrading your {PlatformName} installation, edit the `inventory file so that it matches your desired configuration. You can keep the same parameters from your existing Ansible Automation Platform deployment or you can modify the parameters to match any changes to your environment.
```
Comment (michellemacrh) [inline downstream/modules/platform/proc-update-aap-rpm.adoc:79]: It looks like `inventory is missing a closing symbol

File: downstream/modules/platform/proc-update-aap-rpm.adoc [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+Note: Provide a reachable IP address or fully qualified domain name (FQDN) for all hosts to ensure that users can synchronize and install content from Ansible automation hub from a different node. Do not use localhost. If localhost is used, the upgrade will be stopped as part of preflight checks.
+
+== Backing up RPM-based Ansible Automation Platform
+
+Back up an existing {PlatformNameShort} instance by running the `setup.sh` script with the `backup_dir` flag, which saves the content and configuration of your current environment:
+
+. Go to your Ansible Automation Platform installation directory.
+
+. Run the setup.sh script following the example below:
```
Comment (michellemacrh) [inline downstream/modules/platform/proc-update-aap-rpm.adoc:107]: setup.sh could be wrapped in quotes like it is above `setup.sh`

**Comment by annibond** [comment]
Anni didn't see the suggestions, this should not be pursued further. Donna took care of this in a second ticket.

**Pull request closed without merging**